linters/terragrunt/terragrunt.test.ts

```diff
@@ -0,0 +1,3 @@
+import { linterFmtTest } from "tests";
+
+linterFmtTest({ linterName: "terragrunt" });
```

Comment:
So only the validation subcommand requires terraform to be present? And so we're fine for this subcommand?

Reply:
yep, this one doesn't need terraform